apps/src/StudioApp.js

```diff
@@ -3329,7 +3329,6 @@ StudioApp.prototype.setPageConstants = function(config, appSpecificConstants) {
       is13Plus: config.is13Plus,
       isSignedIn: config.isSignedIn,
       userId: config.userId,
-      verifiedTeacher: config.verifiedTeacher,
```

Comment:
Is there one more place where we can delete this, where the back-end sends this value?